taxi_demo/etl/etl.ipynb

```diff
@@ -0,0 +1,938 @@
+       "                       ...             ...              ...             ...           ...          ...                ...                ...                 ...             ...              ...              ...               ...          ...         ...      ...      ...        ...          ...                   ...          ...                  ...\n",
+       "...                    ...             ...              ...             ...           ...          ...                ...                ...                 ...             ...              ...              ...               ...          ...         ...      ...      ...        ...          ...                   ...          ...                  ...\n",
+       "                       ...             ...              ...             ...           ...          ...                ...                ...                 ...             ...              ...              ...               ...          ...         ...      ...      ...        ...          ...                   ...          ...                  ...\n",
+       "                       ...             ...              ...             ...           ...          ...                ...                ...                 ...             ...              ...              ...               ...          ...         ...      ...      ...        ...          ...                   ...          ...                  ...\n",
+       "Dask Name: concat, 74783 tasks"
+      ]
+     },
+     "execution_count": 22,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "all_taxi = dd.concat(dfs)\n",
+    "all_taxi"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "payment_map = {\n",
+    "    '1': 1,\n",
+    "    'CRD': 1,\n",
+    "    'Credit': 1,\n",
+    "    'Cre': 1,\n",
+    "    'CRE': 1,\n",
+    "    'CREDIT': 1,\n",
+    "    '2': 2,\n",
+    "    'CSH': 2,\n",
+    "    'CASH': 2,\n",
+    "    'Cash': 2,\n",
+    "    'CAS': 2,\n",
+    "    'Cas': 2,\n",
+    "    '3': 3,\n",
+    "    'NOC': 3,\n",
+    "    'No Charge': 3,\n",
+    "    'No': 3,\n",
+    "    '4': 4,\n",
+    "    'DIS': 4,\n",
+    "    'Dispute': 4,\n",
+    "    'Dis': 4,\n",
+    "    '5': 5,\n",
+    "    'UNK': 5,\n",
+    "    'NA': 5,\n",
+    "}\n",
+    "\n",
+    "all_taxi['store_and_fwd_flag'] = all_taxi.store_and_fwd_flag.isin({'1', 'Y'}).astype(int)\n",
+    "all_taxi['payment_type'] = all_taxi.payment_type.map(payment_map).fillna(5).astype(int)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 12.9 s, sys: 253 ms, total: 13.2 s\n",
+      "Wall time: 9min 9s\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%time\n",
```

Comment:
Its not appropriate to persist `all_taxi` before writing because 1) it pretty huge and doesnt fit into memory and 2) were just immediately writing it back out so spending time to persist doesnt gain us much

Timing the repartition/to_parquet is right in this case because I _do_ want to time the whole "pipeline". I can put in some notes that describe that for people new to Dask

Reply:
ah ok then yes, some notes would be helpful. Because Otherwise that timing might be misinterpreted